app/models/card.rb

```diff
@@ -4,12 +4,30 @@ class Card < ActiveRecord::Base
 
   before_validation :set_default_review_date, :strip_whitespaces, on: :create
 
+  scope :random, -> { order("RANDOM()") }
+  scope :need_to_review, -> { where("review_date <= ?", Time.zone.now )
+                              .select(:id, :translated_text)}
```

Comment:
Expression at 9, 61 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 28) spaces for indenting an expression spanning multiple lines.